nengo_viz/static/viz_netgraph.js

```diff
@@ -298,22 +300,40 @@ VIZ.NetGraph.prototype.set_scale = function(scale) {
 
 
 VIZ.NetGraph.prototype.update_font_size = function(scale) {
-    // TODO: text resizing is disabled for now until we can figure out
-    // how best to have it work with large models
+    if (this.zoom_fonts) {
+        $('#main').css('font-size', 3 * this.scale * this.font_size/100 + 'em');
```

Comment:
I'm fine either way... in both cases we'll be putting parameters in somewhere (either here or various .css files).